extensions/adobe/experience/customerJourneyManagement/message-experienceevent.example.1.json

```diff
@@ -0,0 +1,45 @@
+{
+  "xdm:timestamp": "2020-06-15T10:10:00+00:00",
+  "@id": "https://data.adobe.io/experienceid/5ba0475a-5c31-4fce-b573-8ee222a40f45",
+  "xdm:eventType": "message.feedback",
+  "xdm:identityMap": {
+    "ECID": [
+      {
+        "xdm:id": "68519882713298129995549973016107434638",
+        "xdm:primary": true
+      }
+    ]
+  },
+  "xdm:directMarketing": {
+    "xdm:sends": {
+      "xdm:value": 1
```

Comment:
Added explicit measure field for building the reports in CJA. In absence of this explicit fields we have to use the calculated metrics for the send/bounce/click/open metrics.

Reply:
+1